File: services/deid-service/src/main/java/com/researchex/deid/DeidServiceApplication.java
package com.researchex.deid;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/** De-identification 서비스의 애플리케이션 엔트리 포인트. */
@SpringBootApplication
public class DeidServiceApplication {

  public static void main(String[] args) {
    SpringApplication.run(DeidServiceApplication.class, args);
  }
}

File: services/deid-service/src/main/kotlin/com/researchex/deid/DeidServiceApplication.kt
package com.researchex.deid

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/** De-identification 서비스의 애플리케이션 엔트리 포인트다. */
@SpringBootApplication
class DeidServiceApplication

fun main(args: Array<String>) {
    runApplication<DeidServiceApplication>(*args)
}
